src/content/third_party_migration/vendors/redhat-json.json

```diff
@@ -0,0 +1,30 @@
+{
+    "vendor": "redhat-json",
+    "description": "RedHat JSON",
+    "remotes": [
+        {
+            "url": "https://access.redhat.com/labs/securitydataapi/cve.json",
+            "request": "api",
+            "type": "json",
+            "parameters": {
+                "after": {
+                    "type": "fixed",
+                    "description": "Year",
+                    "value": "1999"
+                },
+                "per_page": {
+                    "type": "fixed",
+                    "description": "Vulns per page",
+                    "value": "1000"
+                },
+                "page": {
+                    "type": "variable-incremental",
+                    "value": [
+                        "0",
```

Comment:
- page parameter starts from 1 (this should be contemplated on the schema?)

Reply:
Nice catch! working on it